sympy/tensor/tensor.py

```diff
@@ -1965,7 +1964,7 @@ def tensor_heads(s, index_types, symmetry=None, comm=0):
     return thlist
 
 
-class _TensorMetaclass(ManagedProperties, ABCMeta):
+class _TensorMetaclass(ABCMeta):
```

Comment:
Instead of ABCMeta, this can just subclass from ABC

Reply:
I've changed this to use ABC